python/lsst/pipe/base/graphBuilder.py

```diff
@@ -93,16 +727,24 @@ class GraphBuilder(object):
         Factory object used to load/instantiate PipelineTasks
     registry : `~lsst.daf.butler.Registry`
         Data butler instance.
-    skipExisting : `bool`, optional
-        If ``True`` (default) then Quantum is not created if all its outputs
-        already exist, otherwise exception is raised.
+    skipExisting : `bool`
+        If `True` (default), a Quantum is not created if all its outputs
+        already exist.
+    clobberExisting : `bool`
+        If `True`, overwrite any outputs that already exist.  Cannot be
+        `True` if ``skipExisting`` is.
     """
 
-    def __init__(self, taskFactory, registry, skipExisting=True):
+    def __init__(self, taskFactory, registry, skipExisting=True, clobberExisting=False):
         self.taskFactory = taskFactory
         self.registry = registry
         self.dimensions = registry.dimensions
         self.skipExisting = skipExisting
+        self.clobberExisting = clobberExisting
+
+    ALLOWED_QUANTUM_DIMENSIONS = frozenset(["instrument", "visit", "exposure", "detector",
+                                            "physical_filter", "abstract_filter",
+                                            "skymap", "tract", "patch"])
```

Comment:
Should something like this come from configuration instead? Is this used by anything at all? I looked at this package and `ctrl_mpexec` and could not find any references.

Reply:
Good catch; looks like a relic of an earlier draft.  I'll just remove it.